pages/linux/konsole.md

```diff
@@ -0,0 +1,24 @@
+# konsole
+
+> The KDE terminal emulator.
+> More information: <https://konsole.kde.org>.
+
+- Open a new Konsole in a specific directory:
+
+`konsole --workdir {{path/to/directory}}`
+
+- Run a specific command and do not close the window:
+
+`konsole --noclose -e {{command}}`
```

Comment:
Does `{{command}}` support arguments here? If so, should `{{command}}` be enclosed in `{{token_syntax}}`?

Reply:
could be:
```bash
konsole --new-tab -e /bin/sh -c "cd /media-other/; ./other-program.sh;"
```
all the args after the `-e {{command}}` are applied to the command. so, `-c "cd /media-other/; ./other-program.sh;"` are received by the `sh` command.